llm/throughput/README.md

```diff
@@ -0,0 +1,165 @@
+# MosaicGPT Throughput, MFU, and HFU
+Throughput, MFU, and HFU for different sized MosaicGPT models trained on MCloud.
+MFU and HFU are defined in https://arxiv.org/abs/2205.05198
```

Comment:
Before jumping into all the MFU stuff, can we have a 'Table of Contents' or intro paragraph explaining what the different scripts here are used for? `run_all_config.py` and `parse_logs.py` only make sense in the context of MosaicML Cloud so it would be good to address that